serverlessworkflow/antora.yml

```diff
@@ -6,6 +6,7 @@ nav:
 - "modules/ROOT/nav.adoc"
 asciidoc:
   attributes:
+    kogito_version: 1.23.0.Final
```

Comment:
We don't need this, it should come from the Antora meta info, based on the branch version. @akumar074 is this possible?

Reply:
Ok, I'll wait for @akumar074 before removing it.

Reply:
@ricardozanini @spolti 
We have an intrinsic attribute in Antora `page-component-version`. Its value is defined in `version` key of the `antora.yml`. This would work for our released docs, but for the main branch, we'll have it's value as `main`(as discussed earlier). One way to overcome this is to use something like `1.24.0.SNAPSHOT` in main branch and `1.23.0.Final` for released versions.

Reply:
I think we can do this and use this `version` attribute instead of duplicating it. @radtriste wdyt? This `version` attribute on `antora.yml` impacts the Jenkins pipelines somehow? Do you use this attribute to find/define the branches?

Reply:
in the coming post-release job, we are updating the version component.
so it should be reused throughout the guides directly

Reply:
will it be added to the antora.yaml?
can I remove it from this PR?

Reply:
Jenkins release pipeline will be updating this value: https://github.com/kiegroup/kogito-docs/blob/main/serverlessworkflow/antora.yml#L3
you can reuse it directly. no need for any `kogito_version` attribute

Reply:
and btw, we could use the `2.0.0-SNAPSHOT` into the `main` branch as version, instead of `main`. wdyt @ricardozanini @akumar074

Reply:
@akumar074 @radtriste do you know how can we reusing that field?

Reply:
I think @akumar074 mentioned `page-component-version`

Reply:
worked, thanks!

Reply:
updated.

Reply:
Can you please double-check the other variables if we are not hardcoding the version?

Reply:
didn't find any other usage.